search/src/Components/upload/upload_stepper.js

```diff
@@ -214,10 +247,16 @@ class UploadStepper extends React.Component {
         })
     }
 
+    handleCategoryStatus(category_status){
+        this.setState({category_status: category_status})
```

Comment:
We should probably call this state `categoriesValid` or `categoriesComplete`. Status is not usually a bool, and its interpretation is unclear.

Reply:
categoriesSaved might be better.

Reply:
because status is only true after Apply, I think